app/helpers/time_summary_helper.rb

```diff
@@ -66,4 +66,18 @@ def time_summary_reviews
     Review.list_without_final_review_or_not_closed.
       map { |r| [r.identification, r.id] }
   end
+
+  def time_summary_activities
+    ActivityGroup.list.map do |ag|
```

Comment:
Acá no me di cuenta, pero debería estar ordenado:
```suggestion
    ActivityGroup.list.order(:name).map do |ag|
```